REST_API_DOCS.md

```
@@ -470,26 +464,24 @@ Cmd + T (Ctrl + T) 누르면 탭 추가가 가능합니다. 참고하세요!
-      "post_id": "1",
-      "title": "postname",
-      "author": {
-        "author_id": "admin",
-        "nickname": "abc"
-      },
-      "created_at": "2026-01-07T08:30:00+09:00"
+	    {
+      "post_id" : "1",
+      "title" : "postname",
+      "author" : {
+	      "authorid" : "admin",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)